hoomd/update/box_resize.py

```diff
@@ -99,6 +99,19 @@ def get_box(self, timestep):
         else:
             return None
 
+    @staticmethod
+    def apply(state, box):
```

Comment:
Should we use update? We are using `write` for `Writer`s.

Reply:
Yes, that makes more sense.